Dockerfile

```diff
@@ -14,6 +14,10 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/* \
     && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
+RUN apt-get update
```

Comment:
Seems redundant with line 5

Reply:
I'll try rerunning without it but there's an rm -rf /var/lib/apt/lists which I think wipes out the content of the update.

Reply:
For sure it wipes the update content, but it does this on purpose. The update content should not be saved to the image. 
The `upgrade` should just be in the same line as the `update`
Examples at https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run 

And the changing number of vulnerabilities is probably just that line 17 is busting the cache.